docs/source/experimental_datasets.rst

```diff
@@ -0,0 +1,66 @@
+torchtext.experimental.datasets
+================================
+
+.. currentmodule:: torchtext.experimental.datasets
+
+The following datasets are re-writed and more compatible with ``torch.utils.data``. General use cases are as follows: ::
```

Comment:
nit: do you mean "The following datasets have been rewritten and interact better with ``torch.utils.data``. Here is an example:"?

Reply:
Added to the docs